templates/terraform/examples/monitoring_group_basic.tf.erb

```diff
@@ -1,5 +1,5 @@
 resource "google_monitoring_group" "<%= ctx[:primary_resource_id] %>" {
-  display_name = "<%= ctx[:vars]["display_name"] %>"
+  display_name = "tf-acc <%= ctx[:vars]["display_name"] %>"
```

Comment:
I don't suppose there's a way to make it show as tf-acc in the test but not in the docs?

Reply:
The current `ctx` functionality adds a random suffix in tests but not in docs. We could change it to add the prefix as well. Filing https://github.com/GoogleCloudPlatform/magic-modules/issues/2578 to track.